package.json

```
@@ -40,20 +40,18 @@
     "update:dependency": "update-dependency --lerna",
     "update:versions": "lerna version --no-push --no-git-tag-version"
   },
-  "dependencies": {},
   "devDependencies": {
     "@jupyterlab/buildutils": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "~4.8.1",
-    "@typescript-eslint/parser": "~4.8.1",
-    "eslint": "~7.14.0",
-    "eslint-config-prettier": "~6.15.0",
-    "eslint-plugin-prettier": "~3.1.4",
+    "@typescript-eslint/eslint-plugin": "^5.32.0",
+    "@typescript-eslint/parser": "^5.32.0",
+    "eslint": "^8.21.0",
```

To keep prettier rules:

```suggestion
    "eslint": "^8.21.0",
    "eslint-plugin-prettier": "^4.2.0",
```